app/auth/__init__.py

```
@@ -11,26 +11,17 @@ async def verify_token(request: Request) -> bool:
     bearer_prefix = "Bearer "
 
     if not auth_header or not auth_header.startswith(bearer_prefix):
-        raise HTTPException(
-            status_code=status.HTTP_401_UNAUTHORIZED,
-            detail="Invalid or missing token",
-            headers={"WWW-Authenticate": "Bearer"},
-        )
+        return False  # Invalid or missing token
```

i dont want to merely return.. i want to raise an error.